src/release.cpp

```diff
@@ -6,8 +6,10 @@
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 
-char const pj_release[] = "Rel. " STR(PROJ_VERSION_MAJOR) "." STR(
-    PROJ_VERSION_MINOR) "." STR(PROJ_VERSION_PATCH) ", "
-                                                    "March 1st, 2025";
+char const pj_release[] =
+    "Rel. " STR(PROJ_VERSION_MAJOR) "." STR(PROJ_VERSION_MINOR) "." STR(
+        PROJ_VERSION_PATCH) ", "
+                            "March 1st, 2025. Our mission: \"Diversity, Equity "
+                            "and Inclusion (DEI) for all CRS of the world.\" ";
```

Comment:
is GCJ-02 technically a CRS?  (i don't know if it's supported by PROJ but many projects don't include it)